flake.nix

```diff
@@ -17,7 +17,8 @@
     # when you upgrade `hydra` input remember to also upgrade revs under
     # `source-repository-package`s in `cabal.project`
     hydra = {
-      url = "ssh://git@github.com/input-output-hk/hydra?ref=5ed00dfcd367d0390a774216035e1ea30dde5166";
```

Comment:
I seem to recall we had to use ssh here as it was the only way to get it to clone with submodules, but maybe I am wrong

Reply:
That sounds so absurd that I would expect it to be true in nix land. At least it didn't reject it as invalid, so perhaps it worked? I'll have to find out.